lib/Controller/TrustedSendersController.php

```diff
@@ -83,4 +83,18 @@ public function removeTrust(string $email): JsonResponse {
 
 		return JsonResponse::success(null);
 	}
+	/**
+	 * @NoAdminRequired
+	 * @TrapError
+	 *
+	 *
+	 * @return JsonResponse
+	 */
+	public function list(): JsonResponse {
+		$list =	$this->trustedSenderService->getTrusted(
+			$this->uid,
```

Comment:
```suggestion
			$this->uid
```

trailing commas are only allowed in php7.3+